lib/tasks/exercise.rb

```diff
@@ -1,25 +1,36 @@
 class Exercise
   class << self
     def all
-      slugs.map { |e| new(e) }
+      concept + practice
     end
 
-    private
+    def concept
+      concept_slugs.map { |e| new(e, :concept) }
+    end
+
+    def practice
+      practice_slugs.map { |e| new(e, :practice) }
+    end
+
+    def concept_slugs
+      FileList['exercises/concept/*'].pathmap('%f')
+    end
 
-    def slugs
-      FileList['exercises/*'].pathmap('%f').exclude('TRACK_HINTS.md')
+    def practice_slugs
+      FileList['exercises/practice/*'].pathmap('%f')
     end
   end
```

Comment:
All of this needs DRYing up at some point. But let's get everything working first.